go.mod

```diff
@@ -6,8 +6,8 @@ require (
 	github.com/bramvdbogaerde/go-scp v0.0.0-20201229172121-7a6c0268fa67
 	github.com/container-storage-interface/spec v1.5.0
 	github.com/cucumber/godog v0.12.1
-	github.com/dell/dell-csi-extensions/podmon v1.0.0
-	github.com/dell/gofsutil v1.9.0
+	github.com/dell/dell-csi-extensions/podmon v1.1.1
```

Comment:
Has this podmon version been released? There were no releases planned for dell-csi-extensions

Reply:
Where would I be able to check that? I just ran the automated "check for upgrades" button in vscode and updated the dell libraries.

Reply:
Looks like podmon v1.1.1 is tagged but wasn't released. So lets keep it at the released version v1.0.0

Reply:
Downgrading podmon is causing unit tests to fail and coverage to go down. Not sure why